src/components/TaskBody.vue

```diff
@@ -652,6 +653,27 @@ $breakpoint-mobile: 1024px;
 		opacity: .6;
 	}
 
+	&--non-started .title {
+		color: var(--color-placeholder-dark);
+		font-style: italic;
```

Comment:
Not so sure about the italic, we don't use it very often in Nextcloud. I would tend to remove it.

Reply:
Done !

a191d5e